plugins/modules/packaging/os/homebrew.py

```diff
@@ -168,7 +168,7 @@ class HomebrewException(Exception):
 
 
 # utils ------------------------------------------------------------------- {{{
-def _create_regex_group(s):
+def _create_regex_group_compliment(s):
```

Comment:
You've misspelled [complement](https://en.wikipedia.org/wiki/Complement_(set_theory)) here.  (*compliment* == "Those shoes look nice on you!")

Reply:
Well, it's a nice function, so maybe...? ;)